README.md

```diff
@@ -1 +1,81 @@
-# nanoAOD-tools-modules
\ No newline at end of file
+# Modules for `nanoAOD-tools`
+
+This repo provides some modules that can be used with the
+[`nanoAOD-tools`](https://github.com/cms-nanoAOD/nanoAOD-tools)
+framework for post-processing nanoAOD.
+
+The modules add new branches with like lepton scale factors, using the
+[`correctionlib`](https://github.com/cms-nanoAOD/correctionlib)
+tool, and official
+[JSON correction files](https://gitlab.cern.ch/cms-nanoAOD/jsonpog-integration)
+provided by the CMS POGs.
+
+
+## Installation
+
+### Install CMSSW
+First, install your favorite CMSSW release.
+If you rely on `correctionlib` and `python3`,
+it is strongly recommend to use CMSSW 11.3 or newer.
+For example,
+```
+export SCRAM_ARCH=slc7_amd64_gcc10
+cmsrel CMSSW_12_4_8
+cd $CMSSW_BASE/src/
+cmsenv
+```
+
+### Install `nanoAOD-tools`
```

Comment:
I would somehow make it clearer directly in the subtitles of the README.md that the installation of NanoAOD-tools is required only in case of older CMSSW versions (and discouraged).

something like: 
### Install `nanoAOD-tools` (only necessary for  CMSSW versions < 1X.X)

Reply:
Like this?
https://github.com/IzaakWN/nanoAOD-tools-modules/tree/master#install-nanoaod-tools-for-cmssw-130-or-older

Reply:
Fixed version to CMSSW 12.6:
https://github.com/IzaakWN/nanoAOD-tools-modules/tree/master#install-nanoaod-tools-for-cmssw-126-or-older
Hope this is more clear?

Reply:
Yes thanks

Reply:
Fixed version to "CMSSW 13.2 or older" and "from CMSSW 13.3":
https://github.com/IzaakWN/nanoAOD-tools-modules/tree/master#install-nanoaod-tools-for-cmssw-132-or-older
Hope we're converging on the CMSSW version. 😛